docker-compose.yml

```diff
@@ -0,0 +1,30 @@
+version: "3.7"
+
+services:
+  frontend:
+    image: instapound_frontend
+    container_name: instapound-frontend
+    build:
+      context: ./frontend
+      dockerfile: dev.Dockerfile
+    volumes:
+      - ./frontend:/frontend
```

Comment:
Use `:delegated` for better performance in the dev volumes: `./frontend:/frontend:delegated`